cmake-ide.el

```diff
@@ -657,7 +655,7 @@ the object file's name just above."
     (when (not (file-accessible-directory-p build-dir))
       (cmake-ide--message "Making directory %s" build-dir)
       (make-directory build-dir))
-    (setq cmake-ide-build-dir build-dir)
+;;    (setq cmake-ide-build-dir build-dir)
```

Comment:
?

Reply:
So here if cmake-ide-build-dir is set, and one change cmake-opts or project dir (without setting cmake-ide-build-dir in .dir-locals), then cmake-ide-build-dir will not change accordingly

Reply:
I meant "why is this commented out instead of deleted?"